projects/igniteui-angular/src/lib/combo/combo-dropdown.component.ts

```diff
@@ -87,6 +87,7 @@ export class IgxComboDropDownComponent extends IgxDropDownComponent implements I
     public navigatePrev() {
         if (this._focusedItem && this._focusedItem.index === 0 && this.virtDir.state.startIndex === 0) {
             this.combo.focusSearchInput(false);
+            this.toggle();
```

Comment:
This will lead to the drop-down closing when navigating from the 1st list item w/ `ArrowUp`.
The toggle removed from the `focusSearchInput` is wrapped in a second `if`, so moving it here would change the combo's behavior.